### PR TITLE
Expo--Create expo inside the project

### PR DESCRIPTION
**Pull request by chandantiwari1**
- [x] setup expo app inside the meway
- [x]  setup GitHub action for CI/CD

### CODE REVIEW

File: apps/meway-tablet/eas.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "cli": {
+    "promptToConfigurePushNotifications": false
+  },
+  "build": {
+    "monorepo": {
+      "cache": {
+        "key": "turbo",
+        "customPaths": [
+          "../../node_modules/.cache/turbo"
```
Comment (sutyum) [inline apps/meway-tablet/eas.json:10]: this looks very important, do add a link to a blog post/docs page related to this in the comment here

File: apps/meway-tablet/metro.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Learn more https://docs.expo.dev/guides/monorepos
```
Comment (sutyum) [inline apps/meway-tablet/metro.config.js:1]: another critical page for menorepo, add a reference in the comment

File: apps/meway-tablet/package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@meway/meway-tablet",
+  "version": "0.0.0",
+  "main": "index.ts",
+  "scripts": {
+    "start": "expo start",
+    "dev": "expo start --android",
+    "clean": "rm -rf .expo .turbo node_modules",
+    "web": "expo start --web",
+    "postinstall": "cd ../.. && pnpm build"
```
Comment (sutyum) [inline apps/meway-tablet/package.json:10]: add an explanation here in the comment

File: apps/meway-tablet/src/screens/home.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+import { FlashList } from "@shopify/flash-list";
+
+export const HomeScreen = () => {
+  const [showPost, setShowPost] = React.useState<string | null>(null);
```
Comment (sutyum) [inline apps/meway-tablet/src/screens/home.tsx:14]: why not:
```js
const [showPost, setShowPost] = React.useState<string>("");
```
Is there a reason to include null here?

File: apps/web/package.json [hunk 5]
```diff
@@ -9,10 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@meway/api": "*",
```
Comment (sutyum) [inline apps/web/package.json:12]: add a reference about why this is not `workspace/*`, if you have

File: issue.md [hunk 6]
```diff
@@ -0,0 +1 @@
+how to create checkbox in jquer
```
Comment (sutyum) [inline issue.md:1]: why is this file needed?

File: packages/config/tailwind/global.d.ts [hunk 7]
```diff
@@ -0,0 +1 @@
+declare module "@meway/tailwind-config";
```
Comment (sutyum) [inline packages/config/tailwind/global.d.ts:1]: add a reference as to why this is used or needed

**Review by sutyum — CHANGES_REQUESTED** [review]
This commit represents a lot of hard work. I have left many requests for comments and some minor changes.

File: apps/meway-tablet/eas.json (hunk 1 above)
Comment (chandantiwari1) [reply apps/meway-tablet/eas.json:10]: link to the example github repo: https://github.com/byCedric/expo-monorepo-example
link to the official docs: https://docs.expo.dev/build/eas-json/

File: apps/meway-tablet/metro.config.js (hunk 2 above)
Comment (chandantiwari1) [reply apps/meway-tablet/metro.config.js:1]: official docs link: https://docs.expo.dev/guides/customizing-metro/
reference link: https://github.com/axeldelafosse/expo-next-monorepo-example

File: apps/meway-tablet/package.json (hunk 3 above)
Comment (chandantiwari1) [reply apps/meway-tablet/package.json:10]: Sir we don't need to run postinstall script it's option we can remove it.

File: apps/meway-tablet/src/screens/home.tsx (hunk 4 above)
Comment (chandantiwari1) [reply apps/meway-tablet/src/screens/home.tsx:14]: typescript issue

File: issue.md (hunk 6 above)
Comment (chandantiwari1) [reply issue.md:1]: sir in this files I had to mention all the issues but I forgot

File: packages/config/tailwind/global.d.ts (hunk 7 above)
Comment (chandantiwari1) [reply packages/config/tailwind/global.d.ts:1]: typescript error no type declaration for the @meway file and tailwind file was not workig.

**Review by sutyum — APPROVED** [review]
